examples/README.md

```diff
@@ -0,0 +1,17 @@
+# Python Client Examples
+
+This directory contains various examples how to use the Python client. Please
+read the description at the top of each script for more information about what
```

Comment:
at the top of each example for more information about what the script does and any prerequisites

Reply:
Changed.